Documentation/Assets/Index.rst

```diff
@@ -0,0 +1,70 @@
+.. include:: /Includes.rst.txt
+.. _webroot:
+
+==================
+Webroot and assets
+==================
+
+In TYPO3 files that must be displayed directly in the web browser, may - for
+security reasons - only be placed in certain directories.
+
+In standard Composer-based installation, as this tutorial assumes you have, the webroot
+of the server points to directory :directory:`public`. Read more about this folder:
+:ref:`TYPO3 Explained, folder "public/" <t3coreapi:directory-public>`. Files
+placed outside this folder cannot be called directly by the web browser.
+
+There are roughly 3 types of files tha must be accessed directly by the browser:
+
+*   :ref:`Media and downloads <media>`: Images, Videos and Audio used within the content and
+    manged by editors. Downloads like PDF files, Excel sheets etc.
+*   :ref:`Assets <assets>`: Including CSS, JavaScript, fonts, images and icons used for design purposes.
+*   Technical files managed by Composer and TYPO3, including the `index.php` as
+    Entry point, automatically generated assets and scaled versions of images.
+
+.. _media:
+
+Media and downloads
+===================
+
+Media and download **must** be stored in the so called fileadmin. In standard
+Composer-based installations, as we assume you have here, they are stored in
+:path:`public/fileadmin/`.
+
+Read more about this folder:
+:ref:`TYPO3 Explained, folder "public/fileadmin/" <t3coreapi:directory-public-fileadmin>`.
+
+Files in the fileadmin are managed by the
+:ref:`File abstraction layer (FAL) <t3coreapi:fal_introduction>`.
+
+They can be uploaded, moved and deleted in the backend module
+:guilabel:`File > Filelist` by administrators and
+depending on the :ref:`permissions <permissions-management>`
+by the editors of the site.
+
+..  warning::
+    For security reasons do not store any templates, scripts or assets in this
+    directory. Use a :ref:`site package <t3sitepackage:start>` to store them in
+    :ref:`assets`.
+
+.. _assets:
+
+Assets in extensions and site package
```

Comment:
```suggestion
Assets in extensions and a site package
```

Reply:
Mmh if we think of just one it would probably be "the site package" we could also write site packages...